src/main/java/frc/robot/subsystems/pivot/PivotIOReal.java

```diff
@@ -0,0 +1,97 @@
+package frc.robot.subsystems.pivot;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.trajectory.ExponentialProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.Encoder;
+import frc.robot.Constants;
+
+public class PivotIOReal implements PivotIO {
+
+  private boolean openLoop = false;
+  private double appliedVolts = 0.0;
+
+  private final CANSparkMax motorLeader = new CANSparkMax(69, MotorType.kBrushless);
+  private final CANSparkMax motorFollower = new CANSparkMax(420, MotorType.kBrushless);
+  private final DutyCycleEncoder encoder = new DutyCycleEncoder(1);
+  private final Encoder velEncoder = new Encoder(2, 3);
+  private final ArmFeedforward pivotFFModel = new ArmFeedforward(0.0, 0.0379, 5.85, 0.04);
+  private final PIDController pidController = new PIDController(1, 0, 0); // TODO tune
+  private final ExponentialProfile pivotProfile =
+      new ExponentialProfile(
+          ExponentialProfile.Constraints.fromCharacteristics(10, pivotFFModel.kv, pivotFFModel.ka));
+  private ExponentialProfile.State pivotGoal;
+  private ExponentialProfile.State pivotSetpoint;
+
+  public PivotIOReal() {
+    pivotGoal = new ExponentialProfile.State(1.05, 0);
+    pivotSetpoint = new ExponentialProfile.State(1.05, 0);
+
+    motorLeader.setSmartCurrentLimit(60);
```

Comment:
Magic numbers, especially when repeated, should be defined up above.

Also I feel 60 is a bit high to start